worlds/tits_the_3rd/client/client.py

```diff
@@ -164,27 +171,36 @@ async def give_item(self):
         except IndexError:
             current_item = None
 
+        result = False
+
+        # TODO: Add recipe
         if current_item:
             item_id = current_item.item
-            if item_id is None or item_id >= 500000:  # Special case where we don't actually want to give anything but just acknowledge it
+            # Special case where we don't actually want to give anything but just acknowledge it
+            if item_id is None or item_id >= 500000:
+                result = True
+            # Unlock location
+            elif get_item_id(ItemName.location_min_id) <= item_id <= get_item_id(ItemName.location_max_id):
```

Comment:
Let's use the term "area" instead of "location." Location is confusing because AP already has an alternative meaning for Location, which makes this hard to understand (e.g. unlocking a "location" carries a lot of ambiguity in the context of AP, but unlocking an "area" is straightforward).